### PR TITLE
msg: Format Python scripts

### PR DESCRIPTION
**Pull request by TSC21**
**Describe problem solved by this pull request**
Makes the generation scripts run with Python 3. Enforces *PEP8* stilling. Also, fixes the output of the microRTPS bridge send/receive topic list.

### CODE REVIEW

File: msg/tools/generate_microRTPS_bridge.py [hunk 1]
```diff
@@ -157,40 +156,108 @@ def check_rtps_id_uniqueness(classifier):
...
 parser = argparse.ArgumentParser()
 
-parser.add_argument("-a", "--agent", dest='agent', action="store_true",
-                    help="Flag for generate the agent, by default is true if -c is not specified")
-parser.add_argument("-c", "--client", dest='client', action="store_true",
-                    help="Flag for generate the client, by default is true if -a is not specified")
-parser.add_argument("-i", "--generate-idl", dest='gen_idl',
-                    action="store_true", help="Flag for generate idl files for each msg")
+parser.add_argument(
+    "-a",
```
Comment (bkueng) [inline msg/tools/generate_microRTPS_bridge.py]: Can we have fewer newlines here?

File: msg/tools/generate_microRTPS_bridge.py [hunk 2]
```diff
@@ -281,24 +348,29 @@ def check_rtps_id_uniqueness(classifier):
...
+        else:
             print("Aborting execution...")
             exit(-1)
-        else:
+
+
+    if client:
+        _continue = str(
+            input(
+                "\nFiles in " +
```
Comment (bkueng) [inline msg/tools/generate_microRTPS_bridge.py]: also here

File: msg/tools/generate_microRTPS_bridge.py [hunk 3]
```diff
@@ -337,77 +413,279 @@ def generate_agent(out_dir):
         for msg_file in classifier.msgs_to_send:
             if gen_idl:
                 if out_dir != agent_out_dir:
-                    px_generate_uorb_topic_files.generate_idl_file(msg_file, msg_dir, "", os.path.join(out_dir, "/idl"), urtps_templates_dir,
-                                                                   package, px_generate_uorb_topic_files.INCL_DEFAULT, fastrtpsgen_version, ros2_distro, classifier.msg_id_map)
+                    px_generate_uorb_topic_files.generate_idl_file(
+                        msg_file,
+                        msg_dir,
+                        "",
+                        os.path.join(
```
Comment (bkueng) [inline msg/tools/generate_microRTPS_bridge.py]: and here

File: msg/tools/generate_microRTPS_bridge.py (hunk 1 above)
Comment (TSC21) [reply msg/tools/generate_microRTPS_bridge.py]: This is all `autopep8` doing. I mean I don't mind having the lines broken like this, but if you consider it is better for readability, I can always clean it.

**Comment by TSC21** [comment]
> Can we have a style check in CI?

@bkueng We can. Can that be worked on in a different PR?

**Comment by LorenzMeier** [comment]
I would close this PR, run the styling fixes again on master (which changed since) and get it in. We can then try to polish formatting, but we should not mix up fixes (for Python 3) with cleanups. That always ends up in the place here where we now have a stale PR that is useless, because it has a massive diff and conflicts with master.

**Pull request closed without merging**